src/main.js

```diff
@@ -1,5 +1,5 @@
 const css = 'display:block;display:-webkit-box;-webkit-box-orient:vertical;overflow:hidden;text-overflow:ellipsis'
-const currentValueProp = "data-v-line-clamp-current-value"
+const currentValueProp = "vLineClampValue"
```

Comment:
I chose this attribute's original name very specifically to conform to http://w3c.github.io/html/single-page.html#embedding-custom-non-visible-data-with-the-data-attributes (specifically, it starts with "data-" and it is all lowercase due to the note about automatic lowercasing)

This change you made removes both of those properties. I just wanted to make sure this was intentional and not an oversight.

Thanks for the great library and the fast code reviews!

Reply:
Looks like code reviews are not always that fast :wink: 

Anyway, to conform the w3c you should be using `element.dataset` property as far as I know.
If you add `kebab-case` properties to a DOM element, besides of limiting yourself to use `elem['kebab-case'']` notation (since `elem.kebab-case` throws an error), it doesn't look consistent along all the other properties, see:
![](https://i.imgur.com/Rej44u8.png)

It was intentional indeed, I hope you don't mind that :+1: